src/service/SettingsService.js

```diff
@@ -76,3 +76,19 @@ export const updateAllowNewMailAccounts = (allowed) => {
 	}
 	return axios.post(url, data).then((resp) => resp.data)
 }
+
+export const updateEnabledThreadSummary = async (enabled) => {
+	const url = generateUrl('/apps/mail/api/settings/threadsummary')
+	const data = {
+		enabled,
+	}
+	const resp = await axios.put(url, data)
+	return resp.data
+}
+
+export const isLlmConfigured = async () => {
+	const url = generateUrl('/apps/mail/api/settings/llmconfigured')
```

Comment:
nit: we can save one request at page load if this retrieval is moved into initial state